graph-manager-docs/source/slack-integration.md

```diff
@@ -3,68 +3,75 @@ title: Integrating Graph Manager with Slack
 
-From the Graph Manager, you can configure schema notifications that are triggered when any update, such as type additions, field removals or deprecations, land on your schema. GraphQL makes it extremely easy for developers to collaborate on ever-changing schemas which is why we built this feature to help maintainers and readers of your graph stay in the loop on the latest changes:
+## Schema change notifications
+
+Graph Manager can notify your Slack channel whenever any changes (additions, deprecations, removals, etc.) are made to your graph's registered schema:
 
 <img class="screenshot" src="./img/integrations/schema-notification.jpg" alt="Schema notification Slack message."></img>
 
-To get started, navigate to "Schema Change Notifications" and click on the configure button. You will then be prompted to choose where you want your slack notifications to send to and which variant you want to be notified on. 
+You can configure separate change notifications for each [variant](schema-registry/#managing-environments-with-variants) of your graph.
+
+## Performance alerts
 
-If the channel you want to funnel notifications to isn't set up yet you can simply create a new channel by clicking on the "Slack Channel" drop down and click "New Channel."
+> Performance alerts require a [paid plan](https://www.apollographql.com/pricing/).
 
-You will then need to paste in the Slack hook URL for the channel you want schema change notifications to send to and also give it a name that you can reference to in the Graph Manager.
+You can configure notifications that are triggered on the performance data of your graph, like error percentages and request latencies. This is particularly useful for detecting anomalies, especially around releases. Notifications can be configured to monitor the following metrics for either your entire GraphQL service or individual operations:
 
-<img class="screenshot" src="./img/integrations/slack-integration.jpg" alt="Modal to set up Slack integration"></img>
+- **Request rate:**  requests per minute
+- **Request duration:** p50/p95/p99 service time
+- **Error rate:** errors per minute
+- **Error percentage:** the number of requests with errors, divided by total
+  requests
 
-After that you just need to pick which variant you want to configure notifications for and once you click "Done" you're all set and ready to receive schema change notifiations!
+The triggers you set up are evaluated on a rolling five-minute window. For example, you can configure a notification to trigger when an operation's error rate exceeds 5%. In production, if 6 out of 100 requests result in an error during the last five minutes, the alert will trigger with an error rate of 6%. When the error rate falls back below 5%, your notification will resolve. Here's an example of what the notification looks like:
```

Comment:
[Out of scope] It would be nice to explain that alerts are done for _operation names_ instead of _operation IDs_

Reply:
[Additionally out of scope and also weird] It would be nice to explain that the "All operations" alert actually _excludes_ any operations that you have "overridden" the alert for _for that specific metric_. For instance, if you are alerting on the error percentage of "error percentage" for operation name "foo", the alert you have set up for "error percentage" on "all operations" will be ~all operations~ any operation excluding foo.

Not, uh, celebrating this, but it **is** the state of reality (and out of scope)